src/lib/api/Contacts.ts

```diff
@@ -0,0 +1,27 @@
+import { AxiosInstance } from "axios";
+
+import ContactsApi from "./resources/Contacts";
+
+export default class ContactsBaseAPI {
```

Comment:
There is also a ["get" endpoint for a single Contact](https://github.com/railsware/mailtrap-public-docs/pull/116) underway and will be released soon. I think we should wait for it and add it here.

Reply:
And it was mentioned in the [initial Slack disccussion](https://railsware.slack.com/archives/C05GCAEA6CV/p1748531523508609)